cg-spec/editors_draft.html

```diff
@@ -3031,11 +3033,101 @@ <h2>Alternatives to referential opacity</h2>
+          <ul>
+            <li>For all |t|, |s|, |p|, |o| in the domain IR such that<ul>
+              <li>(|t|, |s|) ∈ IEXT(`unstar:subject`),
+              <li>(|t|, |p|) ∈ IEXT(`unstar:predicate`),
+              <li>(|t|, |o|) ∈ IEXT(`unstar:object`),
+            </ul>
+            <li>and for all terms |s|', |p|', and |o|' such that <ul>
+              <li>|I|(|s|') = |s|,
+              <li>|I|(|p|') = |p|,
+              <li>|I|(|o|') = |o|,
+            </ul>
+            <li>there must exist |t|' in IR such that<ul>
+              <li>(|t|', |s|) ∈ IEXT(`unstar:subject`),
+              <li>(|t|', |p|) ∈ IEXT(`unstar:predicate`),
+              <li>(|t|', |o|) ∈ IEXT(`unstar:object`),
+              <li>(|t|', |I|(L(|s|'))) ∈ IEXT(`unstar:subjectLexical`),
+              <li>(|t|', |I|(L(|p|'))) ∈ IEXT(`unstar:predicateLexical`), and
+              <li>(|t|', |I|(L(|o|'))) ∈ IEXT(`unstar:objectLexical`).
+            </ul>
+          </ul>
+          
+          <li><em>Ensure that TEPs that hold for a triple also hold for all "equivalent" triples.</em>
+          <div>For every |p| such that (|p|,IS(`rdf-star:TransparencyEnforcingProperty`)) ∈ IEXT(IS(`rdf:type`))</div><ul>
+            <li>|p| must be an element of IP
+            <li>For all |t|, |u|, |s|', |p|', |o|', |t|' in IR such that<ul>
+              <li>(|t|, |u|) ∈ IEXT(`p`)
+              <li>(|t|, |s|') ∈ IEXT(`unstar:subject`),
+              <li>(|t|, |p|') ∈ IEXT(`unstar:predicate`),
+              <li>(|t|, |o|') ∈ IEXT(`unstar:object`),
+              <li>(|t|', |s|') ∈ IEXT(`unstar:subject`),
+              <li>(|t|', |p|') ∈ IEXT(`unstar:predicate`),
+              <li>(|t|', |o|') ∈ IEXT(`unstar:object`)
+            </ul>
+            then (|t|', |u|) must also be an element of IEXT(|p|).
+            <li>For all |t|, |u|, |s|', |p|', |o|', |t|' in IR such that<ul>
+              <li>(|u|, |t|) ∈ IEXT(`p`)
+              <li>(|t|, |s|') ∈ IEXT(`unstar:subject`),
+              <li>(|t|, |p|') ∈ IEXT(`unstar:predicate`),
+              <li>(|t|, |o|') ∈ IEXT(`unstar:object`),
+              <li>(|t|', |s|') ∈ IEXT(`unstar:subject`),
+              <li>(|t|', |p|') ∈ IEXT(`unstar:predicate`),
+              <li>(|t|', |o|') ∈ IEXT(`unstar:object`)
+            </ul>
+            then (|u|, |t|') must also be an element of IEXT(|p|).
+          </ul>
+          </ol>          
+          
+        <p>Given two <a>RDF-star graphs</a> |G| and |H|, and following standard terminology, we say that a simple TEP-interpretation |I| satisfies |G| if |I|(|G|) = true, that |G| is (simply) TEP-satisfiable if a simple TEP-interpretation exists that satisfies |G|, otherwise |G| is (simply) TEP-unsatisfiable. We say that |G| simply TEP-entails |H| if every simple TEP-interpretation that satisfies |G| also satisfies |H|. If |G| and |H| each TEP-entail the other, they are TEP-equivalent.</p>
```

Comment:
```suggestion
        <p>Given two <a>RDF-star graphs</a> |G| and |H|, and following standard terminology, we say that a simple TEP-interpretation |I| satisfies |G| if |I|(|G|) = true, and that |G| is (simply) TEP-satisfiable if a simple TEP-interpretation exists that satisfies |G|; otherwise, |G| is (simply) TEP-unsatisfiable. We say that |G| simply TEP-entails |H| if every simple TEP-interpretation that satisfies |G| also satisfies |H|. If |G| and |H| each TEP-entail the other, they are TEP-equivalent.</p>
```